_data/CoolTomatos/CoolTomatos.md

```diff
@@ -0,0 +1,7 @@
+---
+name: CoolTomatos
+institution: Delft University of Technology
+profile_pic: CoolTomatos.jpeg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `CoolTomatos.jpeg` to `CoolTomatos.png` in the markdown file.
2. Rename `CoolTomatos.png` to `CoolTomatos.jpeg`.
I'd suggest you make the change in the markdown file!

Reply:
Hi @Mohammed-Shoaib . Should be changed now. Thanks for catching it!